tests/testthat/helper-mock_bindings.R

```diff
@@ -0,0 +1,4 @@
+# function that can be used with mockery::stub to mock cli messages
```

Comment:
The inclusion of `mockery` :package:  is purely cosmetical so that the output of running the tests doesn't have a bunch of empty lines due to the messages generated by `cli` _(margin on top for `cli_h2`)_.

I can remove it from both PRs if you think this is unnecessary

Reply:
I think we would be good to actually change the tests/functions so they don't output lines instead of importing a new package. Too many imports raises issues of dependency risk.

Reply:
done!